sdks/java/io/kafka/src/main/java/org/apache/beam/sdk/io/kafka/KafkaIO.java

```diff
@@ -729,6 +732,13 @@ public void validate() {
     private Instant curTimestamp;
     private Iterator<PartitionState> curBatch = Collections.emptyIterator();
 
+    private final Counter messagesConsumed = Metrics.counter("io", "messagesConsumed");
+    private final Counter bytesConsumed = Metrics.counter("io", "bytesConsumed");
```

Comment:
So, until we have this API we can have add a utility for common IO related metrics.

For example:
```java
/**
 * Common metrics for sources and sinks.
 */
public class IOMetrics {
  public static Counter readCounter = Metrics.counter("io", "elementsRead");
  ...
}
```

WDYT?